core/templates/domain/topic/TopicObjectFactory.ts

```diff
@@ -633,9 +635,9 @@ export class TopicObjectFactory {
   // the actual topic is fetched from the backend.
   createInterstitialTopic(): Topic {
     return new Topic(
-      null, 'Topic name loading', 'Abbrev. name loading',
+      '', 'Topic name loading', 'Abbrev. name loading',
```

Comment:
No, `''` is not a good replacement for `null`. We should either make sure that we always create the topic with proper values or put null in the places.